code/src/terrat_files/github/sql/insert_repo_tree.sql

```diff
@@ -14,7 +14,7 @@ from unnest(
 ) as x(installation_id, sha, path, changed, id)
 inner join github_installations_map as gim
       on gim.installation_id = x.installation_id
-on conflict on constraint repo_trees_fut_pkey
+on conflict on constraint repo_trees_pkey
```

Comment:
Technically this should go in a different change.  But given the relative little use of tree builder currently, we're going to do it live.